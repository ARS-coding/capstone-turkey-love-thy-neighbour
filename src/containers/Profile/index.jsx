// react
import React from "react";

// react-bootstrap
import { Container } from "react-bootstrap";

function Profile() {
  return (
    <Container>
      <p>Profile</p>
    </Container>
  );
}

export default Profile;
