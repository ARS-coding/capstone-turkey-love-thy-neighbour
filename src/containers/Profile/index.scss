@import "../../style/variables";

.profile-page-bg {
  min-height: calc(100vh - 60px);
  background-image: url("../../images/Profile/PBG.jpg");
  background-repeat: no-repeat;
  background-position: center;
  background-size: cover;

  .profile-content-container {
    min-height: calc(100vh - 60px);

    & > div.row {
      width: 100%;

      & > div:first-child {
        // picture and button column
        padding-right: 0 !important;
        padding-left: 0 !important;

        @media screen and (max-width: 991px) {
          margin-top: 48px;
        }

        @media screen and (min-width: 992px) {
          margin-top: 10px;
        }

        .profile-photo {
          width: 150px;
          height: 150px;
        }
      }

      & > div:last-child {
        // cards column
        padding: 0;

        & > div.cards-row {
          // cards row
          margin: 0;

          & > div {
            // card column
            padding: 0;
            min-height: 250px;

            @media screen and (max-width: 991px) {
              width: 100% !important;
              margin-bottom: 16px;
            }
            @media screen and (max-width: 575px) {
              padding-right: 10px !important;
              padding-left: 10px !important;
            }

            & > div.card {
              border-radius: 15px !important;
              height: 100%;
              margin: 0 !important;

              @media screen and (min-width: 992px) {
                width: 95% !important;
              }
              @media screen and (max-width: 991px) {
                width: 100% !important;
              }

              & > div.card-body {
                & > ul {
                  list-style: none;
                  height: 85%;

                  & > li {
                    color: $logo-blue;
                    font-weight: bold;

                    & > span {
                      color: $black;
                      font-weight: normal;
                    }
                  }
                }

                & > div.card-title {
                  font-weight: bold;
                  margin-left: 30px;
                }
              }
            }

            & > #blue-card {
              background-color: #2a5885;
              color: #ffffff;
              span {
                color: #ffffff;
              }
            }

            & > .white-card {
              background-color: #ffffff;
            }
          }
        }
      }
    }
  }
}
