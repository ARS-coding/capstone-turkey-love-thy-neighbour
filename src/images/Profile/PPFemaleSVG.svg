<svg width="200" height="200" viewBox="0 0 200 200" fill="none" xmlns="http://www.w3.org/2000/svg">
<g clip-path="url(#clip0)">
<path d="M200 100C200.015 112.846 197.549 125.575 192.736 137.485C192.725 137.512 192.716 137.536 192.704 137.562V137.565C189.469 145.54 185.213 153.063 180.044 159.944C179.047 161.275 178.018 162.579 176.958 163.855C176.571 164.322 176.177 164.787 175.784 165.246C173.757 167.598 171.625 169.851 169.387 172.006C168.867 172.509 168.34 173.006 167.807 173.497C167.683 173.612 167.559 173.725 167.435 173.84C164.828 176.222 162.098 178.464 159.254 180.556C157.346 181.967 155.387 183.308 153.378 184.577C152.938 184.858 152.494 185.134 152.047 185.405C151.272 185.879 150.49 186.341 149.701 186.793C148.671 187.385 147.629 187.958 146.574 188.512C146.148 188.74 145.719 188.959 145.29 189.178C144.662 189.5 144.032 189.811 143.396 190.115C143.287 190.169 143.177 190.222 143.068 190.272C142.654 190.473 142.233 190.669 141.813 190.864C136.342 193.387 130.652 195.407 124.813 196.896C124.781 196.905 124.751 196.911 124.719 196.92C124.337 197.018 123.952 197.112 123.568 197.204C123.281 197.275 122.991 197.343 122.701 197.408C122.665 197.42 122.628 197.429 122.591 197.435C121.997 197.574 121.402 197.707 120.801 197.828C120.464 197.899 120.127 197.97 119.79 198.038C119.585 198.083 119.378 198.121 119.174 198.16C118.917 198.21 118.662 198.257 118.405 198.308C118.263 198.337 118.121 198.361 117.976 198.385C117.651 198.444 117.325 198.503 117.003 198.559C116.452 198.654 115.896 198.746 115.343 198.828C114.94 198.891 114.541 198.95 114.139 199.006C113.665 199.074 113.192 199.136 112.716 199.195C112.585 199.216 112.455 199.231 112.325 199.246C112.156 199.266 111.988 199.287 111.819 199.308C111.479 199.349 111.139 199.388 110.796 199.423C110.618 199.444 110.443 199.462 110.266 199.479C109.929 199.515 109.591 199.547 109.254 199.577C108.819 199.618 108.384 199.654 107.946 199.689C107.414 199.731 106.884 199.769 106.349 199.802C106.006 199.822 105.662 199.843 105.316 199.861C104.435 199.908 103.553 199.941 102.665 199.965C102.225 199.976 101.781 199.985 101.337 199.991H101.328C100.887 199.997 100.443 200 99.9996 200C95.5023 200.002 91.0099 199.701 86.5529 199.101C83.2837 198.663 80.0667 198.069 76.902 197.32C76.7748 197.29 76.6476 197.26 76.5233 197.228C76.2038 197.154 75.8872 197.074 75.5706 196.994C75.2896 196.923 75.0115 196.849 74.7304 196.775C67.4868 194.89 60.4796 192.192 53.8428 188.731C53.8103 188.713 53.7748 188.695 53.7422 188.678C53.5499 188.577 53.3606 188.479 53.1712 188.376C52.7541 188.157 52.3389 187.934 51.9257 187.707C49.5736 186.414 47.2817 185.032 45.0499 183.559C44.476 183.183 43.9079 182.802 43.3458 182.414C43.1653 182.29 42.9878 182.166 42.8103 182.041C42.6239 181.911 42.4375 181.781 42.2541 181.651C41.7659 181.305 41.2807 180.955 40.7985 180.601C38.8014 179.133 36.8635 177.594 34.9849 175.982C28.3258 170.279 22.4416 163.73 17.4819 156.5C15.7511 153.979 14.1357 151.378 12.6357 148.695C4.32303 133.814 -0.0280097 117.046 -0.000357295 100C-0.000357295 44.7722 44.7718 0 99.9996 0C155.227 0 200 44.7722 200 100Z" fill="white"/>
<path opacity="0.2" d="M116.08 197.228C116.4 197.154 116.716 197.074 117.033 196.994C117.314 196.923 117.592 196.849 117.873 196.775C117.888 196.186 117.908 195.538 117.929 194.837C118.104 189.346 118.515 180.506 119.485 169.855C119.539 169.284 119.592 168.709 119.645 168.13C119.991 164.521 120.397 160.725 120.882 156.79C120.897 156.66 120.914 156.533 120.929 156.402C121.323 153.213 121.769 149.938 122.272 146.601C122.624 144.266 123.005 141.906 123.414 139.521C123.444 139.355 123.474 139.186 123.503 139.021C123.622 138.331 123.743 137.645 123.867 136.953C124.213 135.027 124.583 133.089 124.971 131.151C126.335 124.334 127.959 117.476 129.897 110.805C131.865 103.886 134.29 97.1046 137.154 90.5059C139.376 85.3798 142.022 80.4485 145.065 75.7633C145.11 75.6953 145.154 75.6302 145.198 75.5621C149.089 69.7012 153.536 64.7278 158.613 61.0089L157.559 59.571C152.695 63.1361 148.234 67.8797 144.175 73.8018C144.154 73.8314 144.136 73.858 144.116 73.8876C141.063 78.414 138.399 83.1904 136.151 88.1657C133.141 94.8207 130.603 101.679 128.553 108.689C126.613 115.204 124.872 122.262 123.332 129.864C122.79 132.541 122.273 135.285 121.781 138.095C121.698 138.574 121.616 139.053 121.533 139.536C121.533 139.541 121.53 139.55 121.53 139.556C121.403 140.296 121.281 141.035 121.157 141.784C120.849 143.669 120.559 145.538 120.287 147.393C119.867 150.251 119.487 153.06 119.148 155.82C119.036 156.731 118.927 157.635 118.823 158.533C118.675 159.799 118.535 161.048 118.403 162.281C118.216 163.997 118.046 165.677 117.891 167.32L117.846 167.772C117.823 168.033 117.799 168.293 117.775 168.55C116.574 181.506 116.198 191.97 116.08 197.228Z" fill="black"/>
<path opacity="0.2" d="M138.76 188.731C138.793 188.713 138.828 188.695 138.861 188.678C139.47 188.361 140.074 188.038 140.677 187.707C140.876 185.346 141.142 182.751 141.491 180.024C141.565 179.462 141.639 178.891 141.722 178.317C141.763 178.009 141.808 177.698 141.852 177.385C142.201 174.982 142.615 172.503 143.112 170.012C145.133 159.846 148.488 149.456 154.059 143.041C154.358 142.695 154.663 142.364 154.979 142.041C155.764 141.226 156.616 140.479 157.527 139.808L156.473 138.373C155.202 139.313 154.032 140.383 152.982 141.565C152.612 141.976 152.251 142.404 151.899 142.849C146.772 149.302 143.571 158.855 141.577 168.314C141.056 170.781 140.618 173.246 140.245 175.645C140.151 176.272 140.06 176.893 139.973 177.509C139.396 181.574 139.015 185.414 138.76 188.731Z" fill="black"/>
<path opacity="0.2" d="M168.562 60.9675C173.478 60.9675 177.464 56.982 177.464 52.0656C177.464 47.1492 173.478 43.1636 168.562 43.1636C163.645 43.1636 159.66 47.1492 159.66 52.0656C159.66 56.982 163.645 60.9675 168.562 60.9675Z" fill="black"/>
<path opacity="0.2" d="M165.695 137.597C170.611 137.597 174.597 133.612 174.597 128.696C174.597 123.779 170.611 119.794 165.695 119.794C160.779 119.794 156.793 123.779 156.793 128.696C156.793 133.612 160.779 137.597 165.695 137.597Z" fill="black"/>
<path opacity="0.2" d="M133.221 67.0182C135.108 77.6683 130.941 87.3117 130.941 87.3117C130.941 87.3117 123.713 79.688 121.826 69.0379C119.938 58.3878 124.105 48.7443 124.105 48.7443C124.105 48.7443 131.333 56.3681 133.221 67.0182Z" fill="black"/>
<path opacity="0.2" d="M162.805 92.2992C152.647 96.0137 142.439 93.63 142.439 93.63C142.439 93.63 148.702 85.2238 158.86 81.5093C169.018 77.7948 179.226 80.1785 179.226 80.1785C179.226 80.1785 172.964 88.5847 162.805 92.2992Z" fill="black"/>
<path opacity="0.2" d="M164.528 160.158C157.408 162.761 150.243 161.063 150.243 161.063C150.243 161.063 154.622 155.143 161.742 152.54C168.863 149.936 176.027 151.634 176.027 151.634C176.027 151.634 171.648 157.554 164.528 160.158Z" fill="black"/>
<path d="M130.166 158.66L130.526 158.846C130.491 159.231 130.405 159.467 130.166 158.66Z" fill="#2F2E41"/>
<path d="M130.642 158.784C130.645 158.825 130.651 158.87 130.654 158.911L130.527 158.846C130.53 158.808 130.533 158.766 130.539 158.725C130.568 158.207 130.533 157.595 130.642 158.784Z" fill="#2F2E41"/>
<path d="M92.2139 144.63C119.045 144.63 140.796 122.879 140.796 96.0476C140.796 69.2162 119.045 47.4651 92.2139 47.4651C65.3825 47.4651 43.6313 69.2162 43.6313 96.0476C43.6313 122.879 65.3825 144.63 92.2139 144.63Z" fill="#2F2E41"/>
<path d="M96.2134 142.65C112.462 142.65 125.635 129.478 125.635 113.229C125.635 96.9796 112.462 83.8072 96.2134 83.8072C79.9644 83.8072 66.792 96.9796 66.792 113.229C66.792 129.478 79.9644 142.65 96.2134 142.65Z" fill="#D0CDE1"/>
<path d="M163.393 169.618L163.358 169.627L156.515 171.269L142.266 174.689L140.322 175.157L118.654 180.355L116.814 180.799L64.2426 193.414C60.6888 192.055 57.2165 190.491 53.8432 188.731C53.8107 188.713 53.7752 188.695 53.7426 188.678C53.5503 188.577 53.361 188.479 53.1716 188.376C52.7544 188.157 52.3393 187.934 51.9261 187.707C49.574 186.414 47.2821 185.032 45.0503 183.559C44.4763 183.183 43.9083 182.802 43.3462 182.414C43.1657 182.29 42.9882 182.166 42.8107 182.041C42.6243 181.911 42.4379 181.781 42.2545 181.651C41.7663 181.305 41.2811 180.955 40.7988 180.601C45.9053 171.32 58.0651 165.151 70.2249 161.775C79.4024 159.225 81.4201 150.843 81.2456 143.385C81.2012 141.683 81.057 139.986 80.8136 138.302C80.4429 135.621 79.8531 132.975 79.0503 130.391L108.473 131.373L111.509 138.367C111.426 139.135 111.399 139.908 111.429 140.68C111.66 146.417 115.201 150.127 119.571 152.521C120.139 152.831 120.722 153.121 121.314 153.391C128.479 156.648 136.914 156.87 136.914 156.87C139.483 156.586 142.077 156.635 144.633 157.015C145.254 157.109 145.858 157.223 146.444 157.355C149.864 158.119 153.095 159.565 155.944 161.607C157.032 162.381 158.061 163.234 159.024 164.16C160.734 165.761 162.206 167.599 163.393 169.618Z" fill="#D0CDE1"/>
<path d="M167.808 173.497C167.683 173.612 167.559 173.725 167.435 173.84C164.828 176.222 162.098 178.464 159.254 180.556C157.346 181.967 155.387 183.308 153.379 184.577C152.938 184.858 152.494 185.134 152.047 185.405C151.272 185.879 150.49 186.341 149.701 186.793C148.671 187.385 147.629 187.958 146.574 188.512C146.148 188.74 145.719 188.959 145.29 189.178C144.663 189.5 144.032 189.811 143.396 190.115C143.287 190.169 143.177 190.222 143.068 190.272L141.816 189.234L156.515 171.269L159.47 167.657C159.47 167.657 160.651 167.657 163.358 169.627C164.854 170.741 166.268 171.961 167.589 173.278C167.66 173.352 167.734 173.423 167.808 173.497Z" fill="#D0CDE1"/>
<path d="M167.6 173.296C167.561 173.482 167.505 173.664 167.435 173.84C164.828 176.222 162.098 178.464 159.254 180.556C156.787 181.446 154.227 182.054 151.624 182.37C151.624 182.37 151.846 183.482 152.047 185.405C151.272 185.879 150.49 186.341 149.701 186.793C148.671 187.385 147.629 187.958 146.574 188.512C146.148 188.74 145.719 188.959 145.29 189.178C144.662 189.5 144.032 189.811 143.396 190.115C143.287 190.169 143.177 190.222 143.068 190.272C142.654 190.473 142.233 190.669 141.813 190.864C136.342 193.387 130.652 195.407 124.813 196.896C124.781 196.905 124.751 196.911 124.719 196.92C124.337 197.018 123.952 197.112 123.568 197.204C123.281 197.275 122.991 197.343 122.701 197.408C122.665 197.42 122.629 197.429 122.591 197.435C121.997 197.574 121.402 197.707 120.802 197.828C120.464 197.899 120.127 197.97 119.79 198.038C119.586 198.083 119.378 198.121 119.174 198.16C118.917 198.21 118.662 198.257 118.405 198.308C118.263 198.337 118.121 198.361 117.976 198.385C117.651 198.444 117.325 198.503 117.003 198.559C116.452 198.654 115.896 198.746 115.343 198.828C114.941 198.891 114.541 198.95 114.139 199.006C113.665 199.074 113.192 199.136 112.716 199.195C112.586 199.216 112.455 199.231 112.325 199.246C112.157 199.266 111.988 199.287 111.819 199.308C111.479 199.349 111.139 199.388 110.796 199.423C110.618 199.444 110.444 199.462 110.266 199.479C109.929 199.515 109.591 199.547 109.254 199.577C108.819 199.618 108.384 199.654 107.947 199.689C107.414 199.731 106.884 199.769 106.349 199.802C106.006 199.822 105.662 199.843 105.316 199.861C104.435 199.908 103.553 199.941 102.665 199.964C102.225 199.976 101.781 199.985 101.337 199.991H101.328C100.887 199.997 100.444 200 99.9998 200C95.5024 200.002 91.0101 199.701 86.553 199.101C83.2838 198.663 80.0668 198.069 76.9021 197.32C76.7749 197.29 76.6477 197.26 76.5235 197.228C76.2039 197.154 75.8874 197.074 75.5708 196.994C75.2897 196.923 75.0116 196.849 74.7306 196.775C67.4869 194.89 60.4798 192.192 53.843 188.731C53.8104 188.713 53.7749 188.695 53.7424 188.678C53.5501 188.577 53.3607 188.479 53.1714 188.376C69.411 184.592 95.0737 178.624 117.343 173.512C117.967 173.37 118.588 173.228 119.204 173.086C127.429 171.198 135.133 169.441 141.651 167.97C142.325 167.82 142.986 167.672 143.633 167.527C150.636 165.953 156.074 164.757 159.023 164.16C159.819 163.978 160.622 163.837 161.432 163.737L167.589 173.278L167.6 173.296Z" fill="#2F2E41"/>
<path d="M47.7663 75.3742C55.7587 75.3742 62.2377 68.8951 62.2377 60.9028C62.2377 52.9105 55.7587 46.4314 47.7663 46.4314C39.774 46.4314 33.2949 52.9105 33.2949 60.9028C33.2949 68.8951 39.774 75.3742 47.7663 75.3742Z" fill="#2F2E41"/>
<path d="M136.662 71.2395C144.654 71.2395 151.133 64.7604 151.133 56.7681C151.133 48.7758 144.654 42.2967 136.662 42.2967C128.669 42.2967 122.19 48.7758 122.19 56.7681C122.19 64.7604 128.669 71.2395 136.662 71.2395Z" fill="#2F2E41"/>
<path d="M121.157 50.5661C121.157 46.9969 122.477 43.5537 124.862 40.8983C127.247 38.2429 130.529 36.5626 134.077 36.1802C132.057 35.9595 130.012 36.167 128.076 36.7893C126.141 37.4116 124.358 38.4348 122.845 39.792C121.331 41.1492 120.121 42.8102 119.292 44.6665C118.463 46.5229 118.035 48.5331 118.035 50.5661C118.035 52.5991 118.463 54.6092 119.292 56.4656C120.121 58.322 121.331 59.9829 122.845 61.3402C124.358 62.6974 126.141 63.7205 128.076 64.3428C130.012 64.9651 132.057 65.1727 134.077 64.9519C130.529 64.5696 127.247 62.8893 124.862 60.2339C122.477 57.5784 121.157 54.1353 121.157 50.5661Z" fill="#2F2E41"/>
<path d="M31.5966 72.6084C29.099 70.0587 27.6318 66.6758 27.4769 63.11C27.3221 59.5441 28.4905 56.0468 30.7577 53.2901C29.1596 54.5468 27.8443 56.1261 26.8974 57.9251C25.9506 59.7241 25.3934 61.7025 25.2621 63.7312C25.1309 65.7599 25.4286 67.7936 26.1358 69.6996C26.843 71.6056 27.9439 73.3412 29.3667 74.7933C30.7895 76.2454 32.5023 77.3815 34.3935 78.1274C36.2846 78.8734 38.3118 79.2125 40.3428 79.1226C42.3738 79.0328 44.3631 78.516 46.181 77.606C47.9989 76.6961 49.6048 75.4132 50.8938 73.8412C48.0914 76.0516 44.571 77.1485 41.0091 76.921C37.4471 76.6934 34.0949 75.1575 31.5966 72.6084Z" fill="#2F2E41"/>
<path d="M92.2137 73.3069C75.0873 73.3069 61.2036 81.4057 61.2036 91.3961C61.2036 95.8662 63.9943 99.9516 68.6009 103.108L74.5707 98.7823L72.9473 105.559C73.6209 105.871 74.3087 106.173 75.0261 106.452L83.9444 96.0476V108.83C86.679 109.268 89.4442 109.487 92.2137 109.485C109.34 109.485 123.224 101.387 123.224 91.3961C123.224 81.4057 109.34 73.3069 92.2137 73.3069Z" fill="#2F2E41"/>
<path d="M122.19 56.7681C122.19 56.7681 119.943 67.1048 133.987 71.2395L122.19 56.7681Z" fill="#2F2E41"/>
<path d="M61.2039 58.6495C61.2039 58.6495 66.3723 70.2058 50.8672 75.3742L61.2039 58.6495Z" fill="#2F2E41"/>
<path d="M68.8686 124.779C70.1725 124.414 70.5194 121.581 69.6433 118.452C68.7672 115.322 66.9999 113.081 65.696 113.446C64.392 113.811 64.0452 116.644 64.9213 119.774C65.7974 122.903 67.5646 125.144 68.8686 124.779Z" fill="#D0CDE1"/>
<path d="M128.572 114.36C128.772 111.116 127.838 108.419 126.487 108.336C125.135 108.253 123.878 110.815 123.678 114.059C123.478 117.302 124.412 119.999 125.764 120.082C127.115 120.166 128.373 117.604 128.572 114.36Z" fill="#D0CDE1"/>
</g>
<defs>
<clipPath id="clip0">
<rect width="200" height="200" fill="white"/>
</clipPath>
</defs>
</svg>
