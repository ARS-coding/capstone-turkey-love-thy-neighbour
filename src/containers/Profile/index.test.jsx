import React from "react";
import renderer from "react-test-renderer";
import configureStore from "redux-mock-store";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import Profile from ".";

describe("Profile page", () => {
    const mockState = {
        user: {
            firestoreDoc: {
                firstName: "Test User",
                gender: "Test Gender"
            },
            authCred: {
                user: {
                    email: "Test Email",
                },
            },
        },
        popup: {
            isEditProfileOpen: false,
        }
    };
    const mockStore = configureStore();
    let store;
    let tree;

    beforeEach(() => {
        store = mockStore(mockState);
        tree = renderer
            .create(
                <Provider store={store}>
                    <Profile />
                </Provider>
            )
            .toJSON();
    });

    it("renders Profile correctly", () => {
        expect(tree).toMatchSnapshot();
    });
});
