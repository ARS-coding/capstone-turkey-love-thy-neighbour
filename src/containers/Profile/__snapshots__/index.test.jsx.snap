// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile page renders Profile correctly 1`] = `
<div
  className="profile-page-bg d-flex align-items-center container-fluid"
  style={null}
>
  <div
    className="profile-content-container d-flex flex-column justify-content-center align-items-center flex-wrap align-content-center container"
  >
    <div
      className="row"
    >
      <div
        className="d-flex flex-column align-items-center mb-3 pe-2 col-sm-12 col-12"
      >
        <div>
          <img
            alt="profile"
            className="profile-photo"
            style={null}
          />
        </div>
        <div>
          <button
            className="sc-bdnxRM sc-dlnjwi fhyKDU dxVdiR"
            onClick={[Function]}
          >
            Edit Profile
          </button>
        </div>
      </div>
      <div
        className="col-sm-12 col-12"
      >
        <div
          className="cards-row d-flex justify-content-around flex-wrap row"
        >
          <div
            className="d-flex justify-content-center align-items-center col-md-4 col-sm-12 col-12"
          >
            <div
              className="info-cards white-card card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title card-title h5"
                >
                  General
                </div>
                <ul
                  className="d-flex flex-column justify-content-around  mb-0"
                >
                  <li>
                    First Name: 
                    <span>
                      Test User
                    </span>
                  </li>
                  <li>
                    Last Name: 
                    <span />
                  </li>
                  <li>
                    Gender:
                     
                    <span>
                      Test Gender
                    </span>
                  </li>
                  <li>
                    Age:
                     
                    <span>
                      Age yet to be added.
                    </span>
                  </li>
                  <li>
                    Education:
                     
                    <span>
                      Education yet to be added.
                    </span>
                  </li>
                  <li>
                    District:
                     
                    <span>
                      District yet to be added.
                    </span>
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div
            className="d-flex justify-content-center align-items-center col-md-4 col-sm-12 col-12"
          >
            <div
              className="info-cards card"
              id="blue-card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title card-title h5"
                >
                  About You
                </div>
                <ul
                  className="d-flex flex-column justify-content-around  mb-0"
                >
                  <li>
                    Bio:
                     
                    <span>
                      Bio yet to be added.
                    </span>
                  </li>
                  <li>
                    Interests:
                     
                    <span>
                      Interests yet to be added.
                    </span>
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div
            className="d-flex justify-content-center align-items-center col-md-4 col-sm-12 col-12"
          >
            <div
              className="info-cards white-card card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title card-title h5"
                >
                  Contact
                </div>
                <ul
                  className="d-flex flex-column justify-content-around  mb-0"
                >
                  <li>
                    Email: 
                    <span />
                  </li>
                  <li>
                    Phone:
                     
                    <span>
                      +90 123 456 78 90
                    </span>
                  </li>
                  <li>
                    Address:
                     
                    <span>
                      Address yet to be added.
                    </span>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
